BlazarUI/app/stylus/main.styl

```diff
@@ -30,6 +30,7 @@
 @import 'components/icon'
 @import 'components/json-modal'
 @import 'components/loader'
+@import 'components/module-modal'
```

Comment:
...thinking about it some more yea i would definitely remove these styles all together and just stick w/bootstrap to keep it simple